pkg/nftables/nftables.tpl

```diff
@@ -8,14 +8,6 @@ table ip firewall {
 		elements = { {{ .InternalPrefixes }} }
 		{{ end }}
 	}
-	# Prefixes in the cluster, typically 10.x.x.x
-	# FIXME Should be filled with nodeCidr
-	set cluster_prefixes {
```

Comment:
can be deleted, still used: 
https://github.com/metal-stack/firewall-controller/blob/master/pkg/nftables/network_policy.go#L74